packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -651,4 +651,16 @@ GridFilterForm.propTypes = {
   valueInputProps: PropTypes.any,
 } as any;
 
+/**
+ *
+ * Demos:
+ *
+ * - [DataGrid](https://mui.com/x/react-data-grid/)
+ * - [DataGridPro](https://mui.com/x/react-data-grid/#pro-plan)
+ * - [DataGridPremium](https://mui.com/x/react-data-grid/#premium-plan)
```

Comment:
What about something more filtering focus?

```suggestion
 * - [Filtering - overview](https://mui.com/x/react-data-grid/filtering/)
```